aten/src/ATen/native/cuda/MultiMarginLoss.cu

```diff
@@ -31,6 +31,7 @@ __global__ void MultiMarginLoss_forward_kernel(
   scalar_t *input_k = input + k*dim;
   scalar_t *output_k = output + k;
   int target_k = static_cast<int>(target[k]);
+  CUDA_KERNEL_ASSERT(target_k >= 0 && target_k < dim);
```

Comment:
can you add an assert message, something about invalid target?

Reply:
Done